cumulusci/tasks/bulkdata/step.py

```diff
@@ -349,11 +352,12 @@ def end(self):
     def load_records(self, records):
         self.batch_ids = []
 
-        for count, csv_batch in enumerate(self._batch(records)):
+        batch_size = self.api_options["batch_size"] or DEFAULT_BULK_BATCH_SIZE
```

Comment:
If you did `self.api_options["batch_size"] = self.api_options.get("batch_size") or DEFAULT_BULK_BATCH_SIZE` on line 336, you could rely on it having a value here.

Reply:
Fixed in e94780b7 and I did the same pattern for REST.